tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -342,6 +342,10 @@ def resource_format(resource, resource_type):
         return "%s" % format_struct(resource)
 
 
+def is_azure_cloud(mailer_config):
```

Comment:
Since you already added an Azure utils, let move this there.

Reply:
I don't think this is the right place.. I think eventually it will be transformed into `get_provider`, so it belongs to the shared code imo

Reply:
Not sure that was obvious to me. Why don't we just implement `get_provider` then?